.github/workflows/applications.yml

```diff
@@ -326,11 +345,12 @@ jobs:
         diff -w <(git show HEAD:cabal.project.freeze) cabal.project.freeze || true
     - name: Sync from cabal cache
       if: env.USE_CABAL_CACHE == 'true'
-      uses: larskuhtz/cabal-cache-action@a0f263d898dd246217960262caa1d381cf066e9a
+      uses: larskuhtz/cabal-cache-action@4b537195b33898fcd9adc62cee2a44986fd7b1b6
       with:
         bucket: "kadena-cabal-cache"
         region: "us-east-1"
-        folder: "${{ matrix.os }}-${{ matrix.ghc }}-${{ matrix.use-freeze-file }}"
+        folder: "packages/${{ matrix.os }}"
+        store_path: ${{ steps.setup.outputs.cabal-store }}
         aws_access_key_id: "${{ secrets.kadena_cabal_cache_aws_access_key_id }}"
         aws_secret_access_key: "${{ secrets.kadena_cabal_cache_aws_secret_access_key }}"
```

Comment:
Frozen builds should use the same cache, because there's actually a chance of sharing builds of dependencies.

The cache handles different ghc versions internally in sub-folders. Also, it uses content based hashes, so there's no risk of conflicts.

Reply:
That makes sense to me.